File: WorkSpace-Web/Classes/Organisation.php
<?php
require_once(realpath(dirname(__FILE__)) . '/OrganisationReservation.php');

use OrganisationReservation;

/**
 * @access public
 * @author creative
 */
class Organisation {
	/**
	 * @AttributeType int
	 */
	private $_idOrganisation;
	/**
	 * @AttributeType String
	 */
	private $_nameOrganisation;
	/**
	 * @AttributeType String
	 */
	private $_responsible;
	/**
	 * @AttributeType String
	 */
	private $_email;
	/**
	 * @AttributeType String
	 */
	private $_password;
	/**
	 * @AttributeType String
	 */
	private $_dateOrganisation;
	/**
	 * @AssociationType OrganisationReservation
	 * @AssociationMultiplicity 0..*
	 */
	public $_reserve = array();

	/**
	 * @access public
	 */
	public function addReservation() {
		// Not yet implemented
	}

	/**
	 * @access public
	 */
	public function observeResevervation() {
		// Not yet implemented
	}

	/**
	 * @access public
	 */
	public function deleteReservation() {
		// Not yet implemented
	}

	/**
	 * @access public
	 */
	public function updateReservation() {
		// Not yet implemented
	}
}
?>

File: Java Classes/Organisation.java
import java.util.ArrayList;

public class Organisation {
	private int idOrganisation;
	private String nameOrganisation;
	private String responsible;
	private String email;
	private String password;
	private String dateOrganisation;
	public ArrayList<OrganisationReservation> reserve = new ArrayList<OrganisationReservation>();

	public void addReservation() {
		throw new UnsupportedOperationException();
	}

	public void observeResevervation() {
		throw new UnsupportedOperationException();
	}

	public void deleteReservation() {
		throw new UnsupportedOperationException();
	}

	public void updateReservation() {
		throw new UnsupportedOperationException();
	}
}